package.nls.json

```diff
@@ -301,7 +301,7 @@
     "AWS.samcli.deploy.stackName.error.length": "A stack name must not be longer than 128 characters",
     "AWS.samcli.deploy.statusbar.message": "$(cloud-upload) Deploying SAM Application to {0}...",
     "AWS.samcli.deploy.template.prompt": "Which SAM Template would you like to deploy to AWS?",
-    "AWS.samcli.configured.location": "Configured SAM CLI Location: {0}",
+    "AWS.samcli.configured.location": "SAM CLI Location: {0}",
```

Comment:
use a more concise message, so that the actual value is more likely to be visible